strings/names/dwarf_first.txt

```diff
@@ -0,0 +1,37 @@
+Thizuil
```

Comment:
I suggest making a .txt of language_DWARF.txt adjectives and have both dwarf names and language pull them out of that txt using regex magic.